util/deadlockhistory/deadlock_history.go

```diff
@@ -183,7 +183,7 @@ func ErrDeadlockToDeadlockRecord(dl *tikverr.ErrDeadlock) *DeadlockRecord {
 		}
 		waitChain = append(waitChain, WaitChainItem{
 			TryLockTxn:     rawItem.Txn,
-			SQLDigest:      sqlDigest,
+			SQLDigest:      hex.EncodeToString(sqlDigest),
```

Comment:
Is it better to be `NewDigest(sqlDigest).String()`?

Reply:
Yes,  use `NewDigest(sqlDigest).String()` is better.